test/run_fail/test_shebang_fail.js

```diff
@@ -0,0 +1,4 @@
+
+#!/usr/bin/env fail
+
+// this should be fail
```

Comment:
missing eof :)

Reply:
linter has been disabled in our test directory, which is not very convenient to detect code flaws and auto-format with linter. It has to be reconsidered how we apply the lint to the tests.